packages/sdk/src/modules/rbac/index.ts

```diff
@@ -766,22 +773,24 @@ export class RBAC extends Base {
 
   /**
    * Fetch all permissions.
-   * @param option - The option for fetch permissions.
+   * @param options - The option for fetch permissions.
    * @returns A promise that resolves when the permissions is fetched.
    */
 
-  public async fetchPermissions(owner: Address): Promise<ResponseFetchGroup[]> {
+  public async fetchPermissions(options: FetchPermissions): Promise<ResponseFetchGroup[]> {
```

Comment:
Returns ResponseFetchGroup, but it is in the fetchPermissions function. It works because fetchPermission has an id, name, and enabled. Should we change the response to a new object only for Permissions?